README.md

```diff
@@ -120,15 +119,40 @@ This library works on "standard" merkle trees designed for Ethereum smart contra
 
 [second preimage attacks]: https://flawed.net.nz/2018/02/21/attacking-merkle-trees-with-a-second-preimage-attack/
 
+## Simple Merkle Trees
+
+The library also supports "simple" merkle trees, which are a simplified version of the standard ones. They are designed to be more flexible and accept arbitrary `bytes32` data as leaves. It keeps the same tree shape and internal pair hashing algorithm.
+
+As opposed to standard trees, leaves are not double-hashed. Instead they are ABI encoded to `bytes32` and hashed in pairs inside the tree. This is useful to override the leaf hashing algorithm and use a different one prior to building the tree.
+
+Users of tooling that produced trees without double leaf hashing can use this feature to build a representation of the tree in JavaScript. We recommend this approach exclusively for trees that are already built on-chain. Otherwise the standard tree may be a better fit.
+
+```typescript
+import { SimpleMerkleTree } from '@openzeppelin/merkle-tree';
+import fs from 'fs';
```

Comment:
We don't use this import explicitelly in the example, so I'd remove it to avoid possible confusion that it must be imported.

```suggestion
```